_pytest/python.py

```diff
@@ -1053,10 +1053,14 @@ def _escape_bytes(val):
            want to return escaped bytes for any byte, even if they match
            a utf-8 string.
         """
-        # source: http://goo.gl/bGsnwC
-        import codecs
-        encoded_bytes, _ = codecs.escape_encode(val)
-        return encoded_bytes.decode('ascii')
+        if val:
+            # source: http://goo.gl/bGsnwC
+            import codecs
```

Comment:
i guess it makes sense to move "import codecs" outside the function (within the py3 if section)